### ISSUE TITLE
How to reproduce this work?

### ISSUE THREAD

**Issue by Raion-Shin** [TIGER-AI-Lab/UniIR#24]
Thanks for your great work! Can you provide more details on the training specifications? Did you use the negative candidates, or how much is the batch size and learning rate?
I tried to trained the CLIP-L Score Fusion on about 32 A100 cards, but the resulting checkpoint gets only recall@5=47% on the EDIS dataset, lower than recall@5=53% of your published checkpoint and recall@5=59.4% that your paper claimed.
